cmd/tealdbg/localLedger.go

```diff
@@ -211,6 +261,34 @@ func getAppCreatorFromIndexer(indexerURL string, indexerToken string, app basics
 	return creator, nil
 }
 
+func getAssetCreatorFromIndexer(indexerURL string, indexerToken string, asset basics.AssetIndex) (basics.Address, error) {
+	queryString := fmt.Sprintf("%s/v2/assets/%d", indexerURL, asset)
+	client := &http.Client{}
+	request, err := http.NewRequest("GET", queryString, nil)
+	request.Header.Set("X-Indexer-API-Token", indexerToken)
```

Comment:
it would be nicer if you could move the ""X-Indexer-API-Token" to a const at the top of the file.